types/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../node_modules/fastify-tsconfig/tsconfig.json",
```

Comment:
https://devblogs.microsoft.com/typescript/announcing-typescript-3-2/#configuration-inheritance-via-node_modules-packages

It looks like TypeScript resolves `node_modules` by default since v3.2